ui/app/src/utils/content.ts

```diff
@@ -165,6 +176,28 @@ export function parseLegacyItemToDetailedItem(item: LegacyItem | LegacyItem[]):
   };
 }
 
+export function parseSandBoxItemToDetailedItem(item: SandboxItem): DetailedItem;
+export function parseSandBoxItemToDetailedItem(item: SandboxItem[]): DetailedItem[];
+export function parseSandBoxItemToDetailedItem(item: SandboxItem | SandboxItem[]): DetailedItem | DetailedItem[] {
+  if (Array.isArray(item)) {
+    // including level descriptors to avoid issues on pathNavigator;
```

Comment:
What’s the issue with level descriptors?

Reply:
the pathNavigator is including levelDescriptor on the list of pages, and the levelDescriptor was not included on the Store